ddtrace/contrib/pymongo/patch.py

```diff
@@ -10,16 +10,15 @@
 
 from ...constants import SPAN_MEASURED_KEY
 from ...ext import SpanTypes
+from ...ext import db
 from ..trace_utils import unwrap as _u
 from .client import TracedMongoClient
 from .client import set_address_tags
 
 
 config._add(
     "pymongo",
-    dict(
-        _default_service="pymongo",
-    ),
+    dict(_default_service="pymongo", dbms_name="mongodb"),
```

Comment:
Is this the dbms_name static or configurable. If it's static it should not live in the config object. We can just define and use a constant. 
```suggestion
    dict(
        _default_service="pymongo",
    ),
```